rules/order.js

```diff
@@ -30,6 +55,7 @@ module.exports = {
         'box-sizing',
         'display',
         'overflow',
+        'vertical-align',
```

Comment:
I don't think it's a fair comparison, most of our code style guidelines (and linting) is stuff that can be learned and remembered pretty easily. You can also often look at surrounding code to find out what the norm is. Neither of those is that simple with CSS props because there are so many of them, and finding one other instance of the prop you're trying to place does not give you all of the information you need to decide precisely where it needs to go to satisfy a linter. You're guessing or reading a detailed style 'spec' for it.

IMO, this is more akin to enforcing ordering groupings and classes in a ruby large helper file based on what type of functionality that helper helps with. Sometimes it's murky and I anticipate there will be a _lot_ of guessing.

I wrote up a car metaphor and then deleted it because that's dumb, but to many, a lot of CSS props are like a [ute](http://s3.caradvice.com.au/thumb/770/382/wp-content/uploads/2015/01/2015-holdenvsford-VFFGX-ute-comparison-23.jpg) in that their categorized purpose is not objective and could easily be viewed multiple ways.

Anyway, sorry for re-booting the original debate inside this arbitrary thread, but I personally don't think it's done with.